### ISSUE TITLE
Wallet menu stops working

### ISSUE THREAD

**Issue by Marec4** [casinocoin/MobileWallet#5]
With fresh install after clicking preferences or transactions, wallet version information is displayed and nothing else on the menu works after that. Also Prypto Cards not working. After generating wallet, sending and receiving some coins these problems disseapeared. Android ver 4.4.2.

**Comment by ajochems** [comment]
Good point, some menu items will only work once a wallet has been created. Actually only the coin, block and exchange info would work as those come from the csc api. I will look into disabling those menu items or maybe better, just open them and show a 'Create a wallet first' message.
